docs/API/Elements.md

```diff
@@ -5,18 +5,60 @@ title: Elements
 
 The Elements API is a powerful abstraction that allows users to manipulate any property on any element through a handful of simple methods. This is important because these few methods allow for easy networking or persistent state. 
 
+To change these properties you will need to reference an object. The most common form to reference an object would be:
+
+```js
+var a = this; 
+```
+You can then use this reference to change other properties.
+```js
+var a = this; 
+a.visible = true; // set object visible to on
+```
+
+### Common properties
+
+Many functions use ID's.
+
+```js
+var a = this.id;   // Read-only. Gets the element's ID
+```
+Any created asset will be a ContentWidget by default. Other possible types include: LightWidget, KinectWidget, ButtonWidget, and TextWidget. Some of these types are only created through scripts. 
+
+```js
+var a = this.type; //Read-only. Get's element type. 
+```
+
+<code>This</code> is a helpful keyword you can use to return most information you will need about your current object.
+
+```js
+function enter() {
+    log.info(this.transform.scale);
+    log.info(this.transform.rotation);
+    log.info(this.transform.position);
+}
+```
+
 ## Hierarchy
 
-An element may have zero to many children, which are simply other Elements, themselves having children. This forms a sort of directed graph, visualized by the tree component in the web editor.
+An element may have one or more children, which are simply other Elements, themselves having children. This forms a sort of directed graph, visualized by the tree component in the web editor.
```

Comment:
Not related to any of your new edits, but I think it's good to point out that the wording here is a bit confusing. I imagine the sentence is trying to indicate that any element can have elements as children, right?

An easier way to word it, if the above is the case, would be the following:

"Any element can have another element as a child. These child elements can also have other elements as children, forming a kind of directed graph. This is visualized by the tree component in the web editor"